### PR TITLE
Implement Pixbuf::copy() and Pixbuf::copy_options()

### LINKED ISSUE #43: Implement gdk_pixbuf_copy/gdk_pixbuf_copy_options
https://developer.gnome.org/gdk-pixbuf/stable/gdk-pixbuf-Image-Data-in-Memory.html#gdk-pixbuf-copy
and
https://developer.gnome.org/gdk-pixbuf/stable/gdk-pixbuf-The-GdkPixbuf-Structure.html#gdk-pixbuf-copy-options

### PR DESCRIPTION
**Pull request by hasufell**
Fixes #43

### CODE REVIEW

File: src/pixbuf.rs [hunk 1]
```diff
@@ -244,4 +246,21 @@ impl Pixbuf {
...
+        unsafe {
+            let copy = ffi::gdk_pixbuf_copy(self.to_glib_none().0);
+            FromGlibPtrFull::from_glib_full(copy)
+        }
+    }
+
+    #[cfg(feature = "v2_36")]
+    pub fn copy_options(&self, dest_pixbuf: &mut Pixbuf) -> Result<(), BoolError> {
+        let err: glib_ffi::gboolean = unsafe {
+            let src = self.to_glib_none().0;
```
Comment (EPashkin) [inline src/pixbuf.rs]: This will be dropped and need moved to function call

Comment (hasufell) [inline src/pixbuf.rs]: I don't quite understand. What will be dropped?

Comment (EPashkin) [inline src/pixbuf.rs]: See https://github.com/gtk-rs/glib/blob/master/src/translate.rs#L35-L40
`self.to_glib_none().0` is alive only on time where `self.to_glib_none()` alive,
in end of this line temporary variable, created by `self.to_glib_none()` is dropped and src points on freed memory.
This why this expression need be moved to function call.

**Comment by hasufell** [comment]
updated

**Comment by EPashkin** [comment]
While rusty way is implementing Clone trait for GdkPixbuf,
I think this 2 functions better as they correspond to each other and documentation.

**Comment by hasufell** [comment]
> While rusty way is implementing Clone trait for GdkPixbuf,

Well, there already is a Clone implementation done by the glib_wrapper macro, which makes it even harder to guess for users what exactly it does. Which is why I think that should be kept consistently and not touched.

**Comment by EPashkin** [comment]
@GuillaumeGomez 👍 for this. Maybe after successful travis rerun

**Comment by GuillaumeGomez** [comment]
Restarted travis.

**Comment by GuillaumeGomez** [comment]
Thanks!

**Pull request merged**